src/minify/minify-css.js

```diff
@@ -13,18 +18,33 @@ function minify(options) {
 
         try {
 
-            var cleaner = new CleanCss({
+            var cleanerInput = {},
+                cleaner;
+
+            cleaner = new CleanCss({
                 advanced: false,
-                restructuring: false
+                restructuring: false,
+                rebase: false,

```

Comment:
Previously we were calling minify on a string of raw code without providing the file path to which the code corresponds. Now that we're generating source maps, we're passing in the file path as well, which means CleanCss is trying to rebase the image URLs based on the CSS file path. We want to maintain the same behavior as before, so I'm just disabling this.